src/External/NBitcoin/ConsensusFactory.cs

```diff
@@ -69,40 +72,14 @@ protected bool IsTransaction<T>()
 
             if (this.IsBlock<T>())
                 result = (T)(object)this.CreateBlock();
-
             else if (this.IsBlockHeader<T>())
                 result = (T)(object)this.CreateBlockHeader();
-
             else if (this.IsTransaction<T>())
                 result = (T)(object)this.CreateTransaction();
 
             return (T)result;
         }
 
-        /// <summary>
-        /// A set of flags representing the capabilities of the protocol.
-        /// </summary>
-        /// <param name="protocolVersion">The version to build the flags from.</param>
-        /// <returns>The <see cref="ProtocolCapabilities"/>.</returns>
-        public virtual ProtocolCapabilities GetProtocolCapabilities(uint protocolVersion)
-        {
-            return new ProtocolCapabilities()
-            {
-                PeerTooOld = protocolVersion < 209U,
```

Comment:
Have not reviewed everything yet, but is this "documentation" added elsewhere?

Reply:
ProtocolCapabilities is from NBitcoin and it was never used really so time to get rid of it